### PR TITLE
Added doble quotes on asterisks and updated some command outputs

### PR DESCRIPTION
**Pull request by perseoGI**

### CODE REVIEW

File: tutorial/creating_packages/create_your_first_package.rst [hunk 1]
```diff
@@ -269,51 +269,51 @@ We can see them with:
...
           revisions
-            dcbfe21e5250264b26595d151796be70 (2024-03-04 17:52:39 UTC)
+            dcbfe21e5250264b26595d151796be70 (2024-05-10 09:40:15 UTC)
               packages
-                6679492451b5d0750f14f9024fdbf84e19d2941b
+                2505f7ebb5a4cca156b2d6b8534f415a4a48b5c9
                   info
                     settings
-                      arch: x86_64
+                      arch: armv8
```
Comment (RubenRBS) [inline tutorial/creating_packages/create_your_first_package.rst:282]: This changes are confusing to the user if they have been following the tutorial, as we have been showing them x86 output (Line 219 for example)

You might want to update those eariler outputs too, or stick with the x86 compilations

**Review by RubenRBS — COMMENTED** [review]
Nice one :)

Minor comment about one of the changes, otherwise looks good!

File: devops/devops_local_recipes_index.rst [hunk 2]
```diff
@@ -138,24 +138,24 @@ We can see now the binary package in our local cache:
...
+          packages
+            72c852c5f0ae27ca0b1741e5fd7c8b8be91a590a
+            info
+              settings
+                arch: x86_64
+                build_type: Release
+                compiler: gcc
+                compiler.version: 9
+                os: Linux
+                options
```
Comment (memsharded) [inline devops/devops_local_recipes_index.rst:156]:
```suggestion
              options
```

File: knowledge/faq.rst [hunk 3]
```diff
@@ -35,7 +35,7 @@ that you get an error like the following one:
     shared=False
 
     ERROR: Missing prebuilt package for 'zlib/1.2.11'. You can try:
-        - List all available packages using 'conan list {ref}:* -r=remote'
+        - List all available packages using 'conan list "{ref}:*" -r=remote'
```
Comment (memsharded) [inline knowledge/faq.rst:38]: Probably this deserves a PR to Conan client too

File: reference/binary_model/package_id.rst [hunk 4]
```diff
@@ -9,33 +9,33 @@ Let's take some package and list its binaries, for example:
     :emphasize-lines: 8,19
 
     $ conan list zlib/1.2.13:* -r=conancenter
```
Comment (memsharded) [inline reference/binary_model/package_id.rst:11]:
```suggestion
    $ conan list "zlib/1.2.13:*" -r=conancenter
```

**Review by memsharded — APPROVED** [review]
Great job

File: reference/binary_model/dependencies.rst [hunk 5]
```diff
@@ -70,16 +70,16 @@ If we now list the ``app/0.1`` binaries, we will see the binary just created:
 
     $ conan list app/0.1:*
```
[the remark this answers is not in the document]
Comment (perseoGI) [reply reference/binary_model/dependencies.rst:71]: Good catch!

File: tutorial/creating_packages/create_your_first_package.rst (hunk 1 above)
Comment (perseoGI) [reply tutorial/creating_packages/create_your_first_package.rst:282]: Yes... I see, maybe I'll update previous commands in order to show the world we are up to date with mac silicon standards

File: reference/binary_model/package_id.rst (hunk 4 above)
Comment (perseoGI) [reply reference/binary_model/package_id.rst:11]: Good catch!

File: knowledge/faq.rst (hunk 3 above)
Comment (perseoGI) [reply knowledge/faq.rst:38]: You mean the help output of `conan list --help`?
```
positional arguments:
  pattern               A pattern in the form 'pkg/version#revision:package_id#revision', e.g: zlib/1.2.13:* means all binaries for zlib/1.2.13. If revision is not specified, it is assumed latest one.
```

I'll update it :)

Comment (memsharded) [reply knowledge/faq.rst:38]: Nope, the error message that produces ``ERROR: Missing prebuilt package for 'zlib/1.2.11'``.

Comment (perseoGI) [reply knowledge/faq.rst:38]: Okey, well in that case both lines should be changed, the list --help and that error. I'll change it

File: tutorial/creating_packages/create_your_first_package.rst (hunk 1 above)
Comment (perseoGI) [reply tutorial/creating_packages/create_your_first_package.rst:281]: I've updated the 219 lines and other in order to be consistent with arm

File: knowledge/faq.rst (hunk 3 above)
Comment (perseoGI) [reply knowledge/faq.rst:38]: Already fixed and merged on conan client

**Review by memsharded — APPROVED** [review]

**Comment by memsharded** [comment]
This PR became outdated pointing to 2.3, I have updated it to 2.4

**Review by RubenRBS — APPROVED** [review]

**Review by czoido — APPROVED** [review]

**Pull request merged**